### ISSUE TITLE
Ingestion again fails for tables with space in name (MSSQL)

### ISSUE THREAD

**Issue by sfirke** [open-metadata/OpenMetadata#4596]
**Affected module**
Ingestion

**Describe the bug**
Regression of #3566.  I have a table called "AVL Data" for instance, here's what the failure looks like in Airflow logs:
```
[2022-04-29 15:29:47,670] {sql_source.py:243} ERROR - Failed to generate sample data for AVL Data - (pytds.tds_base.ProgrammingError) Incorrect syntax near 'limit'.
[SQL: select * from dbo.AVL Data limit 50]
```
On the front end, the table is visible:
![image](https://user-images.githubusercontent.com/7569808/165978166-a3afb1f6-7fdc-4f00-9914-d19bfd3f6e9b.png)

But when I click it I get:
![image](https://user-images.githubusercontent.com/7569808/165978204-de0dae33-c539-4655-8c05-fc25c1a30c7d.png)
![image](https://user-images.githubusercontent.com/7569808/165978229-7cef4945-c03e-4e3e-b5a3-770c5bd5bb42.png)

**To Reproduce**

Try to ingest a table from MSSQL with a space in the name.  I haven't tried it without sample data.

**Expected behavior**
The space to be escaped in the query and the sample data to work.

**Version:**
 - OpenMetadata version: 0.10.0
 - OpenMetadata Ingestion package version: docker compose 2022-04-28

**Additional context**

**Comment by harshach** [comment]
@pmbrull you probably already handling this on the profiler side?

**Comment by ayush-shah** [comment]
Hey @sfirke - 
can you add this `sampleDataQuery` in your sourceConfig as well?



      "sourceConfig": {
            "config": {
              "sampleDataQuery": "select top 50 * from [{}].[{}]"
            }
          }
        },




the error you are facing is related to syntax error, we will provide a default query param, which user can override based on different db services. The above one is thq quick fix for this issue.

**Comment by sfirke** [comment]
Thanks @ayush-shah !  Where would I edit my source config - in the UI, or in a Docker container?  If it's complicated (like in a MySQL db in a Docker container) no need to explain, I can wait on the fix in 0.10.1.
